napari/components/viewer_model.py

```diff
@@ -926,6 +926,116 @@ def add_surface(
         self.add_layer(layer)
         return layer
 
+    def add_swc(
+        self,
+        swc_path,
+        spacing,
+        origin=np.array([0, 0, 0]),
+        *,
+        bounding_box=None,
+        edge_width=1,
+        edge_color='black',
+        face_color='white',
+        z_index=0,
+        name=None,
+        metadata=None,
+        scale=None,
+        translate=None,
+        opacity=0.7,
+        blending='translucent',
+        visible=True,
+    ):
+        """Add a neuron from a swc file as a path shape layer to the layers list.
+
+        Parameters
+        ----------
+        swc_path : str
+            String representing the path to the swc file. Coordinates in swc file are assumed to be in spatial units.
```

Comment:
Looking through napari code base, seems like they have pretty strict line length. These should be broken up so that each line is < 88 characters.